### DISCUSSION TITLE
Different Name for config file

### DISCUSSION THREAD

**Discussion by AbmSourav** [tailwindlabs/tailwindcss#9521] [Ideas]
Now tailwind config file only works with `tailwind.config.js` and `tailwind.js`.
If I use a different name then it doesn't work.
For example: `"tailwindcss tailwindAdmin.config.js -i inputFile -o outputFile`. This doesn't work and also don't give any error.

Expected output: Any file naming should work

**Comment by wongjn** [comment]
If you would like to specify a configuration file that is different to the default name, you can pass the path to it using the `--config` or `-c` flag. In your example, the command might look like:
```sh
tailwindcss -c tailwindAdmin.config.js -i inputFile -o outputFile
```

**Comment by Palmieri78759** [comment]
Thanks for the info I will try to figure it out for more.
[paybyplatema](https://www.paybyplatema.one/)

**Comment by AbmSourav** [comment]
@wongjn Thanks, it works.
But it also raises another problem. If config file name is not `tailwindcss or tailwindcss.config` then I don't get VSCode class name suggestions from TailwindCSS extension.
